cpp/src/parquet/encryption/encryption_internal.h

```diff
@@ -44,7 +45,7 @@ constexpr int8_t kBloomFilterHeader = 8;
 constexpr int8_t kBloomFilterBitset = 9;
 
 /// Performs AES encryption operations with GCM or CTR ciphers.
-class AesEncryptor {
+class PARQUET_EXPORT AesEncryptor {
```

Comment:
Emm why exporting this?

Reply:
Yeah the tests fail to build on Windows without this. I could maybe have tested via the `parquet::encryption::Decryptor` class which is already exported, but the constructor for that needs an `AesDecryptor` so I would still have needed to export these classes anyway, or provide a new method to work around that.